addons/sourcemod/configs/scp_sf/weapons.cfg

```diff
@@ -289,15 +289,15 @@
 	"61"	// Revolver
 	{
 		"classname"	"tf_weapon_revolver"
-		"attributes"	"1 ; 0.760417 ; 6 ; 0.85 ; 43 ; 1 ; 96 ; 1.666667 ; 106 ; 0.25 ; 199 ; 0.9 ; 389 ; 3 ; 4363 ; 0.5"
+		"attributes"	"2 ; 1.5 ; 5 ; 0.8 ; 96 ; 0.2 ; 106 ; 0.25 ; 389 ; 3"
```

Comment:
- Dmg bonus would've been doubled from current (0.76 to 1.5), not sure if thats intended.
- Fire rate would be very slightly faster (0.85 to 0.8), just a 6% increase from current, not sure if thats intended
- While singular reload would be removed, reload time would be 5x faster than stock revolver reload time.

Reply:
Drastic damage buff is intented, as it currently performs worse than starting pistols (such as COM-15)
I did mess up attributes 5 and 96 values however

Reply:
On a second thought, Attribute 43 might infact be pointless when Revolver by default reloads singular.